### PR TITLE
feat(database): add not-null to name and drop title

### PR DESCRIPTION
**Pull request by lvamaral**
what: add a not-null constraint to the name column and drop title column
why: we want to start saving incoming movie titles as name in the DB, and now that we've copied all titles to name we don't need the title column anymore

### CODE REVIEW

**Review by thetonywu — APPROVED** [review]
lgtm

File: lib/plugins/features/movies/controller.js [hunk 1]
```diff
@@ -3,7 +3,12 @@
 const Movie = require('../../../models/movie');
 
 exports.create = (payload) => {
-  payload.name = payload.title;
+  payload = {
+    id: payload.id,
+    name: payload.title,
+    release_year: payload.release_year,
+    object: payload.object
```
Comment (thetonywu) [inline lib/plugins/features/movies/controller.js]: `object` is not needed, it's also not part of the original payload

**Comment by thetonywu** [comment]
whoops, did LGTM prematurely. You'll need to address my comment and fix the failing test

**Comment by lvamaral** [comment]
@tonywuhl everything should be good now!

**Review by jesscxu — APPROVED** [review]
LGTM

**Pull request merged**